www/docs/en/6.x/config_ref/images.md

```diff
@@ -119,6 +120,39 @@ different screen resolutions.
 - [App icon and image size guidelines](https://developer.apple.com/library/ios/documentation/UserExperience/Conceptual/MobileHIG/IconMatrix.html)
 
 ##Windows
+
+For Windows the recommended approach to define app icons is to use `target` attribute.
+
+```xml
+    <platform name="windows">
+        <icon src="res/windows/storelogo.png" target="StoreLogo" />
+        <icon src="res/windows/smalllogo.png" target="Square30x30Logo" />
+        <icon src="res/Windows/Square44x44Logo.png" target="Square44x44Logo" />
```

Comment:
The `src` filename could be any, but the `target` should be one of the icon filenames, used in appxmanifest file. Historically in default cordova-windows template these names are: "StoreLogo", "Square30x30Logo", "Square44x44Logo", "Square70x70Logo", etc...

When copying icons, the resultant filename is calculated as `target + '.' + MRT_qualifiers + extension(src)`